templates/long.yaml

```diff
@@ -0,0 +1,41 @@
+# This expanded template is intentionally kept simple.
+# Each resource type is used as few times as possible.
+# Each use should be for testing different edge cases.
+# Edge cases should be described in the resource name.
+
+- name: Test Project - With Folders
+  type: Project
+  acl:
+  - principal:
+      type: Team
+      id: 3379097
+      name: Sage Bio DREAM Curators
+    principal_id: 3379097
```

Comment:
I think for pre-existing teams and users, we could support `principal_id` field.  That being said, I do like the "principal" object.   One thing that we haven't talked about is the ability to support a list of  users/teams per access type.